### ISSUE TITLE
you kno w theres nothing here besides the read me?

### ISSUE THREAD

**Issue by justfly1111** [dlangille/poudriere_utils#1]
you kno w theres nothing here besides the read me?

**Comment by dlangille** [comment]
Yes, and a statement of "I am waiting for approval to release this" which has not arrived.

**Comment by dlangille** [comment]
I went back looking through old emails to identify what I was going to release. I can't find it.

**Comment by justfly1111** [comment]
was only asking because the repo was two years old and  figured you was either unaware or the project got abandoned ill check in frequently cause am curious

**Comment by dlangille** [comment]
Thank you. I appreciate the inquiry.  Hopefully, it'll come. I'll leave this open.
